dev/Detours/disasm.cpp

```diff
@@ -294,21 +294,21 @@ class CDetourDis
     static BOOL             s_fLimitReferencesToModule;
 
   protected:
-    BOOL                m_bOperandOverride;
-    BOOL                m_bAddressOverride;
-    BOOL                m_bRaxOverride; // AMD64 only
-    BOOL                m_bVex;
-    BOOL                m_bEvex;
-    BOOL                m_bF2;
-    BOOL                m_bF3; // x86 only
-    BYTE                m_nSegmentOverride;
-
-    PBYTE *             m_ppbTarget;
-    LONG *              m_plExtra;
-
-    LONG                m_lScratchExtra;
-    PBYTE               m_pbScratchTarget;
-    BYTE                m_rbScratchDst[64]; // matches or exceeds rbCode
+    BOOL                m_bOperandOverride{ false };
+    BOOL                m_bAddressOverride{ false };
```

Comment:
"bool" goes with "true" and "false", this is C++.
"BOOL" goes with "TRUE" and "FALSE", these are defined by win32.

Reply:
Detours changes are reverted.

I will set it as the baseline for prefast